### DISCUSSION TITLE
Wait time not correctly applied

### DISCUSSION THREAD

**Discussion by menandore** [sn4k3/UVtools#686] [Q&A]
This may be user error (in which case, I apologize), but I haven't been able to convince UVTools to apply the rest time settings correctly.

I've set them to apply whenever values are different and enabled them in settings, but "wait time after cure" isn't showing at all, and "wait time before cure" is set to 0/0, which is well outside the minimum.

I've attached all the relevant files I could think of: the original PWMO, my UVTools settings and suggestions configuration, and my Lychee Slicer scene. My printer is a Photon Mono.
[uvtools.zip](https://github.com/sn4k3/UVtools/files/11125189/uvtools.zip)

![image](https://user-images.githubusercontent.com/5543707/229209107-65379828-f547-4f35-80ca-29b5c9d40176.png)
![image](https://user-images.githubusercontent.com/5543707/229209194-39c21b1c-9a6f-4f40-85e0-9ba82ed55e73.png)

**Comment by sn4k3** [comment]
Anycubic format lacks of per layer wait time and bottom/normal wait time.
The wait time you can set with anycubic is globally set to all layers so UVtools cannot apply the strategy correctly, in that case it will opt for set wait time to the minimum possible to not affect print time, which in your case should be 2s. You got 0s somehow, I will check

**Reply by sn4k3** [reply]
Ok found the problem and will be fixed on next patch. However keep in mind your machine can't benefit from this to effective remove EF

**Comment by menandore** [comment]
Thanks for the quick response!

Understood about the printer - I'm thinking about upgrading anyway, so I'll
keep in mind that anycubic printers can't use some of these tools.
